modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/constants/FDSViewsWebKeys.java

```diff
@@ -10,8 +10,8 @@
  */
 public class FDSViewsWebKeys {
 
-	public static final String FDS_VIEW_ITEM_SELECTOR_DISPLAY_CONTEXT =
-		"FDS_VIEW_ITEM_SELECTOR_DISPLAY_CONTEXT";
+	public static final String DATA_SET_ITEM_SELECTOR_DISPLAY_CONTEXT =
+		"DATA_SET_ITEM_SELECTOR_DISPLAY_CONTEXT";
 
 	public static final String FDS_VIEWS_DISPLAY_CONTEXT =
 		"FDS_VIEWS_DISPLAY_CONTEXT";
```

Comment:
any reason to rename `FDS_VIEW_ITEM_SELECTOR_DISPLAY_CONTEXT` but not `FDS_VIEWS_DISPLAY_CONTEXT` ? maybe bc portlet keeps its name?

Reply:
The module name is still `frontend-data-set-views-web`. I left in naming that derives from module name: `FDSViewsPortlet`, `FDSViewsWebKeys`, `FDSViewsPanelApp` etc. This includes `FDS_VIEWS_DISPLAY_CONTEXT`. The idea is to rename those as a part of move to a standalone app.